tox.ini

```
@@ -0,0 +1,76 @@
+[tox]
+isolated_build = true
+envlist = py36, py37, py38, flake8, mypy, docs
+
+[testenv]
+extras =
+    dev
+commands =
+    pytest -V
+    python -c 'import sys, pkg_resources; dist = pkg_resources.get_distribution("ics"); print(repr(dist), dist.__dict__, sys.path)'
+    pytest --basetemp="{envtmpdir}" {posargs}
+
+[testenv:flake8]
+basepython = python3.8
+commands =
+    flake8 --version
+    flake8 src/
+
+[testenv:mypy]
+basepython = python3.8
+commands =
+    mypy -V
+    mypy --config-file=tox.ini src/
+
```

Why not combine both flake8 and mypy and call them in an additional target `checks` together? The nice thing about this is, you can always amend this new `checks` target with additional tasks. For example, `docformatter` or checking something else. It's also easier to document it

In one of my `tox.ini` files I have this structure (adapted to your existing targets):

```suggestion

[testenv:checks]
description = Run code style checks
basepython = python3
deps =
    {[testenv:flake8]deps}
    {[testenv:mypy]deps}
commands =
    {[testenv:flake8]commands}
    {[testenv:mypy]commands}

```

When you run `tox -e checks` you run both `flake8` and `mypy`. Voila! :wink:

---

That's cool, thanks! Once we got our docstrings to a presentable state, we might even consider having their style checked by a stricter tool like [pydocstyle](http://www.pydocstyle.org/en/stable/), which also checks the docstring contents.